pulseapi/entries/views.py

```diff
@@ -426,6 +426,18 @@ def post(self, request, *args, **kwargs):
             # relation with a Through class, so that needs manual labour:
             creator_data = request_data.pop('creators', None)
 
+            # we also want to make sure that tags are properly split
+            # on commas, in case we get e.g. ['a', 'b' 'c,d']
+            if 'tags' in request_data:
```

Comment:
Question about this: why do we need it if we're handling this at the model save level? Wouldn't the model save function already handle this when the serializer saves?

Reply:
the model save "makes more tags", but has no way to return them for correct crosslinking to "an entry" (the Tag model strictly speaking has no idea what an Entry is), so we need this in two places - in the Tag.save for instances where someone creates a tag directly, and in the POST acceptor for entries so that we can guarantee we start with a preprocessed tag list, so that every tag ends up crosslinked to the to-be-made-right-now Entry.